ios/RNSentry.m

```diff
@@ -77,6 +77,15 @@ + (BOOL)requiresMainQueueSetup {
         reject(@"SentryReactNative", error.localizedDescription, error);
         return;
     }
+    
+    BOOL enableNativeCrashHandling = (BOOL)[mutableOptions valueForKey:@"enableNativeCrashHandling"];
```

Comment:
Is this going to default to `@NO` if there's no value in the key?

Reply:
No, there's always a value, https://github.com/getsentry/sentry-react-native/blob/main/src/js/sdk.tsx#L29

Reply:
I've added a safeguard since one could forcefully `enableNativeCrashHandling: null,`